### PR TITLE
Fix couple of bugs introduced in PR #102

### PR DESCRIPTION
**Pull request by rousik**
PR #102 introduced couple of bugs into the master and was merged to master before the review came in. This PR addresses these issues, specifically:

1. logic for `--clobber` flag has been added
2. fixed logic for --fileType=xls (populating url list, logging errors)
3. use nulls when bad urls are encountered in xls to ensure that the report indexing remains stable.

Sorry for not catching these problems earlier. I should have done more extensive testing of the latest code.

### CODE REVIEW

**Comment by rousik** [comment]
@Winerip for your attention, thanks for the original review.

File: public/tools/auto-lighthouse/index.js [hunk 1]
```diff
@@ -39,23 +39,24 @@ const readWorkbook = async (filepath, resultsPrefix) => {
...
       }
+      urlsToAudit.push(url);
     }
-  } else {
+  } else if (flags.get('inputFormat') == 'text') {
     urlsToAudit = fs.readFileSync(filepath).toString().split("\n");
-    // TODO: throw out things that don't start with http here too?
+    // TODO(rousik): throw out things that don't start with http here too?
+  } else {
+    throw `Uknown input format ${flags.get("inputFormat")}`;
```
Comment (Winerip) [inline public/tools/auto-lighthouse/index.js:59]: Nit: unknown

File: public/tools/auto-lighthouse/index.js [hunk 2]
```diff
@@ -72,8 +73,15 @@ const readWorkbook = async (filepath, resultsPrefix) => {
 
   for (let i=0; i < urlsToAudit.length; i++) {
     const url = urlsToAudit[i];
+    if (url == null) {
+      continue;
+    }
     const resultsFileName = `${RESULTS_FOLDER}/${resultsPrefix}${i}.json`;
 
+    if (fs.existsSync(resultsFileName) && !flags.get('clobber')) {
+      continue;
```
Comment (Winerip) [inline public/tools/auto-lighthouse/index.js:82]: I'd recommend at least logging skipped files.

**Comment by rousik** [comment]
Go ahead with your fixes first and I will resolve and consolidate it with
my code. I won't probably have time for this until at least tomorrow or
Monday morning.

Sorry for all the trouble :)

**Comment by Winerip** [comment]
Don't worry about the issues. I assume this should be good to resume after merging.

A couple other things I think the script should do - if the script could check that the requestedUrl and finalUrl fields are the same (to make sure there weren't any redirects, which can happen on pages behind authentication) and make sure runWarnings is empty (I think there's a chance of transient errors there) that would probably help, especially if the script is being run overnight. You can integrate these, or I can I add them in after this is merged.

**Comment by rousik** [comment]
I wonder whether some of these things could be addressed as a final
processing stage between collection and rendering on the React app,
specifically we should be dealing with things like:
1. non-200 http responses (as the audits will be for 404 page instead of
the requested one)
2. deduplication by finalUrl (partially overlaps with (1))

We could also use this to build set of urls for which we have audits and
this could be used to rerun the entire audit in an automated fashion.

I wonder whether we would like to do this checking for individual batches.

**Pull request merged**